build.gradle

```diff
@@ -45,6 +45,9 @@ allprojects {
     repositories {
         mavenCentral()
```

Comment:
We shouldn't require mavencentral and others

Reply:
hi @ashoktelukuntla those changes already part of data-prepper root of main branch. we have not made any sunch changes.

Reply:
Thanks for getting back on this.